packages/polyfills/package.json

```diff
@@ -24,15 +24,16 @@
   },
   "homepage": "https://github.com/Shopify/quilt/blob/master/packages/polyfills/README.md",
   "dependencies": {
-    "@babel/polyfill": "^7.10.1",
     "@shopify/useful-types": "^2.2.1",
     "browser-unhandled-rejection": "^1.0.2",
     "caniuse-api": "^3.0.0",
+    "core-js": "^3.6.5",
```

Comment:
Why isn't v3 begin reflected in the yarn.lock 🤔

Reply:
Probably because other packages use v2? I haven't really looked much into that as it seems upgrading to v3 is more problematic than I thought